example.config.js

```diff
@@ -42,6 +42,12 @@ module.exports = {
 		 * Pass false to disable this feature.
 		 */
 		report: -1001148607297,
+		/**
```

Comment:
I'd add a blank line between settings.
Also could put it before `report` to have them in alphabetic order :P